org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/debug/XtextBuildConsole.xtend

```diff
@@ -24,7 +28,10 @@ class XtextBuildConsole extends IOConsole {
 	val PrintStream out
 
 	new() {
-		super("Xtext Build", "xtextBuildConsole", null, true)
+		super("Xtext Build", "xtextBuildConsole",
+			ImageDescriptor.createFromURL(
+				FileLocator.find(Platform.getBundle("org.eclipse.xtext.builder"), new Path("icons/console.png"), null)),
```

Comment:
`Activator.^default.bundle.getEntry("icons/console.png")` should do the job also